src/components/SharedComponents/Buttons/INatIconButton.js

```diff
@@ -77,9 +77,12 @@ const INatIconButton = ( {
       onPress={onPress}
       style={( { pressed } ) => [
         {
-          opacity: pressed
-            ? 0.95
-            : 1,
+          // eslint-disable-next-line no-nested-ternary
+          opacity: disabled
+            ? 0.5
+            : pressed
+              ? 0.95
+              : 1,
```

Comment:
Is there a reason to ignore the rule here? I guess I'm on the fence about nested ternaries. Sometimes I find them hard to read, sometimes not.